pkg/types/types.go

```diff
@@ -12,9 +12,10 @@ type Collection struct {
 	CreateTypes  map[string]string `json:"createTypes,omitempty"`
 	Actions      map[string]string `json:"actions"`
 	ResourceType string            `json:"resourceType"`
-	Pagination   *Pagination       `json:"pagination,omitempty"`
 	Revision     string            `json:"revision,omitempty"`
 	Continue     string            `json:"continue,omitempty"`
```

Comment:
Is `Continue` also an old pagination variable? I didn't see it being used in apiserver, but I am not sure about our other projects.

Reply:
It's not old, it's part of the response from k8s when the result is partial, see https://kubernetes.io/docs/reference/using-api/api-concepts/#retrieving-large-results-sets-in-chunks

Reply:
Thanks for showing me that, I was wondering why it was a string.